rules/cross-platform/impact_hosts_file_modified.toml

```
@@ -22,7 +22,7 @@ references = ["https://www.elastic.co/guide/en/beats/auditbeat/current/auditbeat
 risk_score = 47
 rule_id = "9c260313-c811-4ec8-ab89-8f6530e0246c"
 severity = "medium"
-tags = ["Elastic", "Linux", "Windows", "macOS"]
+tags = ["Elastic", "Host", "Linux", "Advanced Threat Detection", "Impact", "Windows", "macOS"]
```

should we move Windows and macOS up?
```suggestion
tags = ["Elastic", "Host", "Linux", "Windows", "macOS", "Advanced Threat Detection", "Impact"]
```

---

Yes, working on improving ordering for multiple `Host` platforms now.